Server/Resources.xcassets/springboard-alerts/springboard-alerts-el.dataset/alerts.json

```diff
@@ -499,29 +500,61 @@
     "buttons": [
       "Να επιτρέπεται πάντα",
       "Να επιτρέπεται",
-      "Κατά τη χρήση της εφαρμογής"
+      "Κατά τη χρήση της εφαρμογής",
+      "Να επιτραπεί"
     ],
     "shouldAccept": true
   },
   {
     "title": "Να επιτραπεί σε «%@» η χρήση της τοποθεσίας σας ακόμη κι όταν δεν χρησιμοποιείτε την εφαρμογή;",
     "buttons": [
-      "Να επιτρέπεται"
+      "Να επιτρέπεται",
+      "Να επιτραπεί"
     ],
     "shouldAccept": true
   },
   {
     "title": "Να επιτραπεί επίσης στην εφαρμογή «%@» η χρήση της τοποθεσίας σας ακόμη κι όταν δεν χρησιμοποιείτε την εφαρμογή;",
     "buttons": [
-      "Να επιτρέπεται"
+      "Να επιτρέπεται",
+      "Να επιτραπεί"
     ],
     "shouldAccept": true
   },
   {
     "title": "Αίτημα από «%@» για εύρεση και σύνδεση σε συσκευές στο τοπικό σας δίκτυο.",
+    "buttons": [
+      "OK",
+      "ΟΚ"
```

Comment:
Do we really need to keep duplicated buttons?

Reply:
Looks like these to "OK" were typed in different letters (Latin and Cyrillic probably)
Could you please confirm?